PhysicsTools/PatAlgos/python/slimming/slimmedElectrons_cfi.py

```diff
@@ -26,3 +26,6 @@
 
 from Configuration.Eras.Modifier_bParking_cff import bParking
 bParking.toModify(slimmedElectrons,dropCorrections="0")
+bParking.toModify(slimmedElectrons,dropIsolations="0")
+bParking.toModify(slimmedElectrons,dropShapes="0")
+bParking.toModify(slimmedElectrons,dropClassifications="0")
```

Comment:
```suggestion
bParking.toModify(slimmedElectrons,dropCorrections="0", dropIsolations="0", dropShapes="0", dropClassifications="0")
```
this is simpler to maintain if we need to extend to another modifier

Reply:
@jainshilpi 
are you available to update this PR?

Reply:
Hi Slava, I missed that comment. Just pushed the modification.

Reply:
> Hi Slava, I missed that comment. Just pushed the modification.

Thank you.